### PR TITLE
Translate Hebrew + Support RTL

### PR DESCRIPTION
**Pull request by moshfeu**
Thanks for very handy extension!
This PR contains translations for hebrew (I'm native speaker).
It also contains support RTL so in RTL languages (currently only Hebrew) the extension will present the content currectly.
If it's too much and you want only the translations, let me know.

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -121,6 +121,12 @@ cme.onUninstalled.addListener(id => {
 	$('#' + escapeCssId(id)).remove();
 });
 
+const uiLang = chrome.i18n.getUILanguage();
+const rtlLangs = ['he'];
+$('html')
+	.attr('lang', uiLang)
+	.toggleClass('rtl', rtlLangs.includes(uiLang));
```
Comment (fregante) [inline src/popup.js:128]: Does this work? So the rtl languages aren't hardcoded
```suggestion
$('html').attr({
	lang: chrome.i18n.getUILanguage(),
	dir: chrome.i18n.getMessage('@@bidi_dir')
});
```

File: src/popup.css [hunk 2]
```diff
@@ -188,4 +188,16 @@ button:hover {
...
+ * Right to Left
+ */
+html.rtl {
+  direction: rtl;
+  text-align: right;
+}
+
+html.rtl button {
+  text-align: right;
 }
```
Comment (fregante) [inline src/popup.css:203]:
```suggestion
html[dir='rtl'] {
  text-align: right;
}

html[dir='rtl'] button {
  text-align: right;
}
```

I think the `dir` attribute also automatically sets the direction. Maybe it sets _all_ of this. Can you check if it works without this CSS?

**Comment by fregante** [comment]
Thank you for your PR! Can you test my suggested changes?

File: src/popup.js (hunk 1 above)
Comment (moshfeu) [reply src/popup.js]: I was happy to learn about `chrome.i18n.getMessage('@@bidi_dir')` 😀

**Comment by fregante** [comment]
I think the dir attribute also automatically sets the direction. Maybe it sets all of this. Can you check if it works without this CSS?

File: src/popup.js [hunk 3]
```diff
@@ -121,6 +121,13 @@ cme.onUninstalled.addListener(id => {
 	$('#' + escapeCssId(id)).remove();
 });
 
+$('html')
+	.attr({
+		lang: chrome.i18n.getUILanguage(),
+		dir: chrome.i18n.getMessage('@@bidi_dir')
+	})
+	.get(0).style.setProperty('--align-to', chrome.i18n.getMessage('@@bidi_start_edge'));
```
Comment (fregante) [inline src/popup.js:129]: Why do this? This should just follow the `dir` attribute and the variable can be set via CSS

Comment (moshfeu) [reply src/popup.js]: Because I wasn't read few more lines 😅

File: src/popup.css [hunk 4]
```diff
@@ -1,4 +1,9 @@
 /* RESETS */
+:root {
+  --ext-icon-margin: 0.3em;
+  --align-to: __MSG_@@@@bidi_start_edge__;
```
Comment (fregante) [inline src/popup.css:4]: Oh this is better than what I had in mind!

File: src/popup.css [hunk 5]
```diff
@@ -78,7 +83,7 @@ button {
   line-height: 1.9;
   font-weight: bolder;
   border-bottom: 1px solid #88888833;
-  padding-right: var(--margin);
+  padding: 0 var(--margin) 0 0;
```
Comment (fregante) [inline src/popup.css:86]: Doesn't this meant `padding-top`? Also where is `--margin` defined?

File: src/popup.css [hunk 6]
```diff
@@ -105,15 +110,15 @@ button {
   flex-grow: 1;
   overflow: hidden;
   text-overflow: ellipsis;
-  padding-left: var(--margin);
+  padding: 0 0 0 var(--margin);
```
Comment (fregante) [inline src/popup.css:113]: This means `padding-right`

File: src/popup.css [hunk 7]
```diff
@@ -105,15 +110,15 @@ button {
...
 }
 .ext img {
   width: 16px;
   height: 16px;
   vertical-align: -3px;
 }
 .extIcon {
-  margin-right: 0.3em;
+  margin: 0 var(--ext-icon-margin) 0 0;
```
Comment (fregante) [inline src/popup.css:121]: This means `margin-top`

File: src/popup.css (hunk 5 above)
Comment (moshfeu) [inline src/popup.css:86]: I'm not sure why by the syntax `padding-__MSG_@@bidi_start_edge__: var(--margin);;` is not working for me.
The [docs](https://developer.chrome.com/extensions/i18n#:~:text=Here's%20an%20example%20of%20using) says that it should.

File: src/popup.css (hunk 7 above)
Comment (moshfeu) [reply src/popup.css:121]: I'm not sure about it (unless I'm missing something of course)

https://jsbin.com/wozogotiqi/edit?html,css,output

Screen Shot 2020-04-07 at 23 16 38

Comment (fregante) [reply src/popup.css:121]: D'oh.

**Comment by moshfeu** [comment]
Is there anything else that needs to be done?

**Comment by hankxdev** [comment]
@fregante is this PR good to merge?

**Comment by fregante** [comment]
Logical CSS to the rescue: https://rtlstyling.com/posts/rtl-styling/#logical-properties-cheat-sheet

rtl
ltr

**Pull request merged**